examples/step-18/step-18.cc

```diff
@@ -993,8 +987,11 @@ namespace Step18
                     eps_phi_i = get_strain(fe_values, i, q_point),
                     eps_phi_j = get_strain(fe_values, j, q_point);
 
-                  cell_matrix(i, j) += (eps_phi_i * stress_strain_tensor *
-                                        eps_phi_j * fe_values.JxW(q_point));
+                  cell_matrix(i, j) += (eps_phi_i *             //
+                                        stress_strain_tensor *  //
+                                        eps_phi_j               //
+                                       ) *                      //
+                                       fe_values.JxW(q_point)); //
```

Comment:
The tester says that this is an extra parenthesis.

Reply:
ha, true. Will fix.